lib/src/locale/chinese_hans.dart

```diff
@@ -1,7 +1,7 @@
 part of duration.locale;
 
-class ChineseDurationLocale implements DurationLocale {
-  const ChineseDurationLocale();
+class ChineseSimplifiedDurationLocale implements DurationLocale {
```

Comment:
This would mean a breaking change. Is it possible to keep the old name here? Or would that be wrong?

Reply:
I don't mind keeping the same name but why would this be a breaking change? Isn't it only being used internally in the plugin?